service/awsconfig/v4/resource/cloudformation/spec.go

```diff
@@ -15,10 +15,6 @@ const (
 
 	workerRoleKey = "WorkerRole"
 
-	cloudFormationGuestTemplatesDirectory    = "service/templates/cloudformation/guest"
-	cloudFormationHostPreTemplatesDirectory  = "service/templates/cloudformation/host-pre"
-	cloudFormationHostPostTemplatesDirectory = "service/templates/cloudformation/host-post"
```

Comment:
All this is completely gone with the general rendering. Note that we have go imports for this as well now for v4.